src/classes/Controllers/Web/Servers.php

```diff
@@ -46,7 +46,7 @@ public function render(Request $request, Response $response, array $args): array
 	{
 		$stable_version = $this->simplecache->get('stable_version');
 		if (!$stable_version) {
-			$stable_version = trim(file_get_contents('https://git.friendi.ca/friendica/friendica/raw/branch/master/VERSION'));
+			$stable_version = trim(file_get_contents('https://git.friendi.ca/friendica/friendica/raw/branch/stable/VERSION'));
```

Comment:
Should I create a stable branch, branced off from the current master branch, for this to work already or is this only preparation?

Reply:
The stable branch forked from master already exists.

Reply:
But it should be pushed to the repositories so that the directory can find the information right?

Reply:
Yes.

Reply:
Never mind, it already exists: https://github.com/friendica/friendica-directory/tree/stable

I thought you checked and that I just created it on my own fork.

Reply:
But the code references the Friendica repository, not the directory repository.

Reply:
Right, sorry, let me create it.